src/systems/systems_wasm/src/lib.rs

```diff
@@ -93,8 +103,7 @@ impl<'a, 'b> MsfsSimulationBuilder<'a, 'b> {
         mut self,
         aircraft_ctor_fn: U,
     ) -> Result<(Simulation<T>, MsfsHandler), Box<dyn Error>> {
-        let mut aspects: Vec<Box<dyn SimulatorAspect>> =
-            vec![Box::new(self.electrical_buses.unwrap())];
+        let mut aspects: Vec<Box<dyn Aspect>> = vec![Box::new(self.electrical_buses.unwrap())];
         if self.apu.is_some() {
             aspects.push(Box::new(self.apu.unwrap()));
         }
```

Comment:
I'm aware that this is not directly related to this PR, but I am wondering why are the APU and the electrical busses treated differently from other `MsfsAspect`s. Why are they not built from and `MsfsAspectBuilder` like the brakes for example?

Reply:
Well spotted. While working on this I realised that probably the APU and electrical buses can be replaced by what I built here as well. However, I did not want to increase the scope of the PR, so I'll make those changes in a later PR.